text/3771-demote-i686-pc-windows-gnu.md

```diff
@@ -0,0 +1,174 @@
+
+A blog post will be made to describe the change.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+By no longer doing automated testing for this target, this target will likely deteriorate more quickly than with continued automated testing.
+
+Additionally, this opens the door for further demotions in the future, like removing host tools, which could still be useful to some people.
+But such demotions will always have to be justified on their own.
+
+# Rationale and alternatives
+[rationale-and-alternatives]: #rationale-and-alternatives
+
+The maintenance requirement violation can be solved by multiple people stepping up to maintain this target. This has not happened so far.
+
+The popularity requirement could be fulfilled by more people using this target, but this does not seem possible, as 32-bit x86 as been on a decline for a long time, as new CPU models for this architecture are no longer being made.
+
+# Prior art
+[prior-art]: #prior-art
+
+This is the first time [since the Target Tier Policy was created](https://rust-lang.github.io/rfcs/2803-target-tier-policy.html) (note that this links to an old version, see [the rustc book](https://doc.rust-lang.org/1.84.1/rustc/target-tier-policy.html) for the latest version at the time of writing) that a Tier 1 target is being demoted.
+
+Before that, there has been the [demotion of `i686-apple-darwin` from Tier 1 to Tier 3 in 2019](https://github.com/rust-lang/rfcs/pull/2837).
+The reasoning there was mostly Apple's support being removed, which is not the case here.
+The measures in this RFC are much less drastic.
+
+The [promotion of `aarch64-apple-darwin` to Tier 1](https://github.com/rust-lang/rfcs/pull/3671) cited popularity as the major motivation, matching unpopularity as one of the motivations here.
+
+This is a continuation of [MCP 822](https://github.com/rust-lang/compiler-team/issues/822), which contains some additional details in the description and linked Zulip stream.
+
+# Unresolved questions
+[unresolved-questions]: #unresolved-questions
+
+None so far.
+
+# Future possibilities
+[future-possibilities]: #future-possibilities
+
+`x86_64-pc-windows-gnu` will remain a Tier 1 target after this RFC.
+While its popularity is more aligned with Tier 1, it suffers from the same lack of maintenance (but to a lesser degree) as its 32-bit cousin.
+It may be demoted as well in the future.
+
+`i686-pc-windows-gnu` may be demoted to a Tier 2 target without host tools in the future if it is not deemed useful enough.
+This will likely happen in the near future, but is not part of this RFC.
+That demotion will not need an RFC.
+
+The `*-windows-gnullvm` targets, which are based on LLVM instead of GNU tools, may see increased maintenance and popularity in the future, replacing the `*-windows-gnu` targets.
+But it seems unlikely that `i686-pc-windows-gnullvm` would ever acquire Tier 1 status.
```

Comment:
```suggestion
(However, it is unclear whether `i686-pc-windows-gnullvm` would get enough interest and support to attain Tier 1 status.)
```

Reply:
It's straight up impossible.
LLVM uses memory mapping for many IO operations. On 32-bit Windows, it can not even host itself due to insufficient address space.

Reply:
@mati865 Is that an issue with the compiler or with the linker? (Would it work if linked with a different linker?)

In any case, thanks for the clarification. I'll edit the suggestion.